### PR TITLE
Update installing Ruby page (id)

### PR DESCRIPTION
**Pull request by meisyal**
Changes:

* Improve OpenIndiana installation section as mentioned on commit dc375e4aa909ce78d9cfb019a4a81940086830cf
* Drop mention of specific Ruby versions (see PR #1681)
* Remove Solaris section from installation page (see PR #1682)
* Clean up and update Installation guide (see PR #1877)
* Add `snap` installation method (see PR #1888)

### CODE REVIEW

**Comment by meisyal** [comment]
Just updated with adding WSL on installation method (see PR #2009).

**Comment by meisyal** [comment]
cc: @gozali

File: id/documentation/installation/index.md [hunk 1]
```diff
@@ -154,73 +200,69 @@ Informasi lebih lanjut terkait Ruby dan ekosistem sekelilingnya pada FreeBSD
...
 
 
 ### Distribusi Lainnya
 {: #other-systems}
 
 Pada sistem lain, Anda dapat mencari *package repository* dari
-*manager* distribusi Linux Anda untuk Ruby, atau kakas bantu pihak ketiga mungkin
-pilihan tepat untuk Anda.
+*manager* distribusi Linux Anda untuk Ruby. Kemungkinan lain, Anda dapat
+menggunakan sebuah [third-party installer][installers].
```
Comment (kuntoaji) [inline id/documentation/installation/index.md]: I think it's better to use italic for this sentence.

`[*third-party installer*][installers]`

Comment (meisyal) [reply id/documentation/installation/index.md]: OK. Just changed that sentence to italic, @kuntoaji.

**Comment by meisyal** [comment]
Thanks @kuntoaji

**Pull request merged**